cni-plugin/Makefile

```diff
@@ -140,13 +141,31 @@ $(DEPLOY_CONTAINER_FIPS_MARKER): Dockerfile build fetch-cni-bins
 	$(MAKE) retag-build-images-with-registries VALIDARCHES=$(ARCH) IMAGETAG=latest-fips LATEST_IMAGE_TAG=latest-fips
 	touch $@
 
+CN_FLAGS=-ldflags "-X github.com/containernetworking/plugins/pkg/utils/buildversion.BuildVersion=$(GIT_VERSION)"
+
+CONTAINERNETWORKING_PLUGINS_CREATED=.containernetworking-plugins-$(ARCH).created
+CONTAINERNETWORKING_PLUGINS_CLONED=.containernetworking-plugins.cloned
+
+$(CONTAINERNETWORKING_PLUGINS_CLONED):
+	git clone --single-branch --branch $(CNI_VERSION) https://github.com/projectcalico/containernetworking-plugins.git
+	touch $@
+
+$(CONTAINERNETWORKING_PLUGINS_CREATED): $(CONTAINERNETWORKING_PLUGINS_CLONED)
```

Comment:
Not sure it's any cleaner(!) but there's a way to tell make that this target makes several binaries on a single invocation:
```
.../bin/foo .../bin/bar .../bin/baz &: $(TOUCHFILE)
    ...
```
The `&:` means "one call to this creates all these files"

I used that in the BPF builds to avoid issues when calling through to the BPF makefile.

Reply:
I took over your suggestion. I think it does look slightly cleaner now. Thanks for teaching me this feature.